nzedb/Music.php

```diff
@@ -126,8 +131,7 @@ public function getMusicInfoByName($artist, $album)
 			$album = preg_replace('/( - | -|\(.+\)|\(|\))/', ' ', $album);
 			$album = preg_replace('/[^\w ]+/', '', $album);
 			$album = preg_replace('/(WEB|FLAC|CD)/', '', $album);
-			$album = trim(preg_replace('/\s\s+/i', ' ', $album));
-			$album = trim($album);
+			$album = trim(trim(preg_replace('/\s\s+/i', ' ', $album)));
```

Comment:
This looks like a mistake to me. Shouldn't the first trim() have done all that's needed so the second one is a mistake that should be removed, not collapsed into the previous statement.